libusb/io.c

```diff
@@ -2755,15 +2752,12 @@ const struct libusb_pollfd ** LIBUSB_CALL libusb_get_pollfds(
  * Since version 1.0.20, \ref LIBUSB_API_VERSION >= 0x01000104
  *
  * It is legal to call this function with a NULL pollfd list. In this case,
- * the function will simply return safely.
+ * the function will simply no nothing.
```

Comment:
I guess a "return" word is missing here.

Reply:
Whoops, I meant "do". :)

Reply:
Fixed.